feature/steps/create_account.py

```diff
@@ -0,0 +1,18 @@
+from behave import when, then
+from feature.page_objects.base_page import click_element_by_css_selector, send_keys_by_css_selector, wait_until_alert_is_present, get_alert_text, get_uuid4
+
+UUID = get_uuid4()
+
+@when('the user complete the Sign Up form')
+def complete_sign_up_form(context):
+    click_element_by_css_selector(context, '#signin2')
+    send_keys_by_css_selector(context, '#sign-username', UUID)
+    send_keys_by_css_selector(context, '#sign-password', UUID)
```

Comment:
I know that this are just "test" users, but it would not be a good practice to create a user wich password is the same string as the user. for several reasons, you should have a constant for that (can be a string you can remember, to use for every user)